test/static-code

```diff
@@ -39,6 +39,11 @@ test_pyvulture() {
         --ignore-decorators '@*'
 }
 
+test_mypy() {
+    command -v mypy >/dev/null || skip 'no mypy'
+    mypy --no-error-summary test/common/testlib.py
```

Comment:
Is there any danger we'll get py2 `mypy` here?  Does `python3 -m mypy` work, like for the other things?

Reply:
Hmmmm it seems Debian does not provide python2 mypy at all https://packages.debian.org/buster/python3-mypy

Reply:
In python-dbusmock I call mypy through `python3 -m mypy`, which is generally the recommended and more robust way of invoking these tools. The only known exception that I know is flake8.